app/controllers/saved_scenarios_controller.rb

```diff
@@ -114,10 +126,53 @@ def update
     end
   end
 
+  # Soft-deletes the scenario so that it no longer appears in listings.
+  #
+  # PUT /saved_scenarios/:id/discard
+  def discard
+    unless @saved_scenario.discarded?
+      @saved_scenario.discarded_at = Time.zone.now
+      @saved_scenario.save(touch: false)
+
+      flash.notice = 'Your scenario was put in the trash'
```

Comment:
Would you mind if we put up some translations for the notices as well? Having it in English here in between all the Dutch makes it look somehow unfinished to me. If you like the idea I can provide the translations 🙂 

<img width="957" alt="Screenshot 2022-05-20 at 15 57 46" src="https://user-images.githubusercontent.com/14875123/169545695-182cf075-1156-4c79-a619-f845d7eaf4fd.png">

Reply:
I created the first version in English only and thought I had caught all these. Apparently not!

I moved them into the locale files. There are [two more messages to be translated](https://github.com/quintel/etmodel/blob/bfd847da4669a2414291a37f049de2785625a1a3/config/locales/nl_etm.yml#L257-L258), please.